README.md

```diff
@@ -391,8 +414,25 @@ optimism_package:
       # Available services:
       # - blockscout
       # - rollup-boost
+      # - da_server
```

Comment:
should we remove this and literally force all chains to spin up a da-server if altda deploy is set to true?